projects/js-packages/components/components/alert/index.tsx

```diff
@@ -1,3 +1,4 @@
+// @ts-check
```

Comment:
You don't need to add `// @ts-check` here, this file is already in `Typescript`.

`// @ts-check` is made to be used in `.js{x}` files if you want them to force `Typescript` check.

We won't use this right now.

Reply:
🤦 rebased